server/src-lib/Hasura/Server/Init.hs

```diff
@@ -47,7 +47,7 @@ newtype PGVersion = PGVersion { unPGVersion :: Int } deriving (Show, Eq, J.ToJSO
 
 getDbId :: Q.TxE QErr Text
 getDbId =
-  (runIdentity . Q.getRow) <$>
+  runIdentity . Q.getRow <$>
```

Comment:
I prefer the original in terms of readability but I don't feel particularly strongly about it.

Reply:
I think hlint's reasoning is that if you shouldn't use parentheses if you don't need them. This happens when the operators automatically bind as desired.

Reply:
As discussed in our meeting today, I reinstated these particular parentheses and disabled the hint for now until we make up our mind about a more precise plan.